src/query/expression/src/evaluator.rs

```diff
@@ -89,6 +104,16 @@ impl<'a> Evaluator<'a> {
         }
     }
 
+    pub fn run_exprs(&self, exprs: &[Expr]) -> Result<Vec<BlockEntry>> {
```

Comment:
should return `Result<Vec<Value<AnyType>>>`, keep consistent with `run` function.

Reply:
seems we don't need this function, better to call `run` function outside.

Reply:
thx, fixed